docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -0,0 +1,207 @@
+{
```

Comment:
<p>Since it's a rectangle should we return a BBOX not a polygon? Actually this polygon would be invalid if copy/pasted anyways because the first and last point are not the same (5 points).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
honestly my preferred solution is that the value is available as a variable in notebook code cells. I'm poking around to see if I can figure out how to make that happen